packages/GA4Client/src/commerce-handler.js

```diff
@@ -270,7 +277,11 @@ function parseProduct(_product) {
         }
     }
 
-    return product;
+    if (self.common.forwarderSettings.enableDataCleansing) {
```

Comment:
I think doing it in the log event logic complicates it in its own way.  It would be something like. 
 
```
// loop over each item first, then another loop for item attributes such as: 
for (itemAttr in product.items) {
   itemAttr = this.standardizeParameters(ittemAttr);
}
```

But maybe a zoom will be easier to discuss in real time.